Assets/Scripts/UI/MouseOverRoomDetails.cs

```diff
@@ -53,11 +54,12 @@ void Update()
 
         string s = "";
 
-        foreach (string g in t.Room.GetGasNames())
+        foreach (string gasName in t.Room.GetGasNames())
         {
-            s += g + ": " + t.Room.GetGasPressure(g) + " (" + (t.Room.GetGasPercentage(g) * 100) + "%) ";
+            s += gasName + ": " + "(" + t.Room.ChangedGases(gasName) + ") ";
+            s += string.Format("{0:0.000}", t.Room.GetGasPressure(gasName)) + " atm";
+            s += " (" + string.Format("{0:0.0}", t.Room.GetGasFraction(gasName)*100) + "%)\n";
```

Comment:
string.Format("{0}: ({1}) {3} atm ({4}%", gasName, t.Room...) If mean line this :-)